app/views/users/edit.html.erb

```diff
@@ -1,12 +1,29 @@
 <% provide(:title, t(".edit_user")) %>
-<% provide(:button_text, t(".save_change")) %>
 <h1><%= t ".update_profile" %></h1>
+
 <div class="row">
   <div class="col-md-6 col-md-offset-3">
-    <%= render "form" %>
+    <%= form_for(@user) do |f| %>
+      <%= render "shared/error_messages", object: f.object %>
+
+      <%= f.label :name %>
+      <%= f.text_field :name, class: "form-control" %>
+
+      <%= f.label :email %>
+      <%= f.email_field :email, class: "form-control" %>
+
+      <%= f.label :password %>
+      <%= f.password_field :password, class: "form-control" %>
+
+      <%= f.label :password_confirmation, t(".confirm") %>
+      <%= f.password_field :password_confirmation, class: "form-control" %>
+
+      <%= f.submit t(".save_change"), class: "btn btn-primary" %>
+    <% end %>
+
     <div class="gravatar_edit">
       <%= gravatar_for @user %>
-      <%= link_to t(".change"), Settings.edit_user, target: "_blank" %>
```

Comment:
sao đang dùng Rails built-in helper `link_to` ngon nghẻ, em lại chuyển qua dùng code HTML thuần nhỉ? :p

Reply:
e đã sửa lại rồi ạ